samples/SortBenchmark/SortBenchmark.fs

```diff
@@ -43,40 +43,23 @@ open Prajna.Tools
 open Prajna.Tools.FSharp
 open Prajna.Tools.StringTools
 open Prajna.Core
+//open Prajna.Service.CoreServices
+open Prajna.Api.FSharp
 open Prajna.Api.FSharp
-
 
 open sortbenchmark
 open Microsoft.FSharp.NativeInterop
-open Prajna.Service.FSharp
+//open Prajna.Service.FSharp
 
 let Usage = "
     Usage: Benchmark performance for distributed sort. \n\
     Command line arguments:\n\
     -in         Copy into Prajna \n\
-    -out        Copy outof Prajna \n\
     -dir        Directory where the sort gen file stays \n\
-    -num        Number of remote instances running \n\
-    -sort       Executing Sort (1: 1-pass sort, 2-2 pass sort) \n\
-    -nump N     Number of partitions \n\
+    -nump       Number of partitions in the second stage repartition\n\
```

Comment:
We should start using a proper command line parsing library. Ken was working in VMHub with the same one we used in TLC/AzureML Algos. It's pretty good. I'm sure there are others. This should not block this pull, of course, as there's tons of code that's like this and we don't need to change it now. But going forward, we should move towards something like that.